### ISSUE TITLE
Aria-pressed not triggering in DOM

### ISSUE THREAD

**Issue by Venrir** [sampotts/plyr#1058]
Preview Gif:
![](https://venipa.net/u/2018-06-20/gJlB5UTYQYvDWO.gif)

### Expected behaviour
aria-pressed should be called on pressing obviously.

### Actual behaviour
aria-pressed not triggering always stays default (false/true).
play button always stays with the Play icon

### Environment

- Browser: Edge, Firefox, Chrome
- Version: 3.3.17 (NPM)
- Operating System: Windows 10

### Steps to reproduce
- Laravel Webpack Mix (require plyr there)

**Comment by jwjcmw** [comment]
This looks like the same thing as #1062

**Comment by sampotts** [comment]
Closing in favour of #1062
